spec/index.bs

```diff
@@ -1545,30 +1545,34 @@ interface GPUMapMode {
 
   **Returns:** {{ArrayBuffer}}
 
+  1. If |size| is unspecified, let |rangeSize| be this.{{[[size]]}}. Otherwise, let |rangeSize| be |size|.
```

Comment:
I think this line need to take into account the offset so an undefined size means "the rest of the buffer"

Reply:
I was going for that but then I read
>optional GPUSize64 size, undefined means buffer.size, 0 means 0?

which confused me a bit. He probably meant buffer.size - offset here. I'll change it then, as it makes more sense that way.